src/odemis/acq/drift/__init__.py

```diff
@@ -244,20 +249,23 @@ def estimateCorrectionPeriod(period, t, repetitions):
 
         return itertools.cycle(acq_dc_period)
 
-    def _updateSEMSettings(self):
+    def _updateBeamSettings(self):
```

Comment:
Indeed, "SEM" is not well fitting anymore. However, "beam" usually refers to settings like voltage, current, etc. So I'd suggest `_updateScannerSettings()`.